README.md

```diff
@@ -1 +1,42 @@
-# Sygma Widget UI
\ No newline at end of file
+# Sygma Widget UI
+
+The Sygma Widget is a customizable frontend that leverages the `Sygma protocol` and can be integrated into any Dapp regardless of the framework used. More information can be found in our [docs](https://docs.buildwithsygma.com/)
+
+This repository is divided into two packages. The [Widget](./packages/widget/) package is a web component built using Lit framework that allows you to have a widget for the `Sygma` protocol project`. The [React](./packages/react/) package is a wrapper around the Lit Widget that allows developers to use this application inside react projects.
+
+## Quick setup
+
+```bash
+yarn create vite my-dapp --template react-ts
+cd ./my-dapp
```

Comment:
first time playing around with react. bear with me on any beginner questions/issues @wainola 😅 
once i cd into my-dapp, i get a usage error, which i get around by `touch yarn.lock` inside my-dapp. after that i am able to follow the rest of this setup. 
not sure if this is a commonly known thing for dependency management